### PR TITLE
Bug fix for incorrect UI Text

### LINKED ISSUE #310: [PE-D][Tester E] Ambiguity on unique wedding name behaviour
From the wording of "wedding names are case sensitive", I assumed that names with different capitalisations would mean that they are considered as different weddings, but they are treated as the same wedding.

-------------
Labels: `type.FunctionalityBug` `severity.Low`
original: kohkakohla/ped#6

### PR DESCRIPTION
**Pull request by HanB1n**
Wedding names is supposed to be case insensitive but the UI text reflects as case sensitive.

Fixes #310

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/wedding/CreateWeddingCommand.java [hunk 1]
```diff
@@ -21,7 +21,7 @@ public class CreateWeddingCommand extends Command {
     public static final String COMMAND_KEYWORD = "cw";
 
     public static final String MESSAGE_USAGE = COMMAND_WORD + ": Creates a wedding in the address book. "
-            + "Wedding names are case sensitive.\n"
+            + "Wedding names are incase sensitive.\n"
```
Comment (dasha3412) [inline src/main/java/seedu/address/logic/commands/wedding/CreateWeddingCommand.java:24]: Maybe typo, should be case insensitive?

**Review by dasha3412 — CHANGES_REQUESTED** [review]
Small typo

**Review by dasha3412 — APPROVED** [review]
LGTM!

File: src/main/java/seedu/address/logic/commands/wedding/CreateWeddingCommand.java (hunk 1 above)
Comment (HanB1n) [reply src/main/java/seedu/address/logic/commands/wedding/CreateWeddingCommand.java]: yup

**Pull request merged**